spec/factories/tasks.rb

```diff
@@ -1,12 +1,22 @@
 FactoryBot.define do
   factory :task do
-    # 下記の内容は実際に作成するカラム名に合わせて変更してください
     title { 'test_title_01' }
```

Comment:
`create_list(:task, n)`での作成に耐えられるよう、`sequence`を利用してunique制約があっても回避できるようにした方が良いです。
```suggestion
    sequence(:title) { |n| "title_#{n}" }
```

Reply:
b5d4917　で修正しました。